src/backend/wallet_connect.nim

```diff
@@ -31,6 +38,33 @@ proc addSession*(sessionJson: string): bool =
     warn "AddWalletConnectSession failed: ", "msg", e.msg
     return false
 
+proc disconnectSession*(topic: string): bool =
+  try:
+    let rpcRes = disconnectWalletConnectSession(topic)
+    return isSuccessResponse(rpcRes):
+  except Exception as e:
+    warn "wallet_disconnectWalletConnectSession failed: ", "msg", e.msg
+    return false
+
+proc getActiveSessions*(validAtTimestamp: int): JsonNode =
+  try:
+    let rpcRes = getWalletConnectActiveSessions(validAtTimestamp)
+    if(not isSuccessResponse(rpcRes)):
+      return nil
+
+    let jsonResultStr = rpcRes.result.getStr()
+    if jsonResultStr == "null":
+      return nil
+
+    if rpcRes.result.kind != JArray:
+      error "Unexpected result kind: ", rpcRes.result.kind
+      return nil
+
+    return rpcRes.result
+  except Exception as e:
+    warn "GetWalletConnectActiveSessions failed: ", "msg", e.msg
```

Comment:
Maybe `error` instead of `warn`? Same question for `disconnectSession`, `safeSignTypedData`, `signMessage`